base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle

```diff
@@ -0,0 +1,24 @@
+asciidoctorj {
+    version '2.0.0'
+    modules{
+        diagram{
+            version '1.5.16'
```

Comment:
Last available version is 1.5.18

Reply:
Thanks @abelsromero. By other hand in gradle I'm using the plugin 3.0.0-alpha.2 althought alpha.3 is available. I tested both of them and for basic docs they work fine but my intent is to upgrade the profile when we'll have a release of the plugin